setupToolbox.m

```diff
@@ -0,0 +1,3 @@
+% One-line script which adds /toolbox folder contents (including subfolders) to path for current MATLAB session
+% Key use case: enables specific live scripts in /examples folder opened to via Open in MATLAB Online badge to utilize library code (i.e., in /toolbox folder), for simple toolboxes (no dependencies) not currently using package management. 
+addpath(genpath("../../toolbox"));
```

Comment:
Considering that the path to the toolbox root folder is hardcoded, the setupToolbox script MUST have a pre-specified location within the code. You may want to describe it here and/or in the README.

Reply:
Good point! I've tried to clarify this, how does it seem?